#### iNetModels 2.0: an interactive visualisation and database of multi-omics data
By: Muhammad Arif et al.
BiorXiv

To get a comprehensive understanding of the biological processes underpinning human disease, it is essential to reveal associations between different omics data. iNetModels is hence an interactive database and visualisation platform of Multi-Omics Biological Networks (MOBNs). 

The platform allows the exploration of associations between clinical chemistry, anthropometric parameters [[202118011002-Anthropometric_Parameters]], plasma proteomics and metabolomics, as well as metagenomics [[202118011003-Metagenomics]] for oral and gut microbiome obtained from the same individual. It also includes tissue- and cancer-specific Gene Co-expression Networks (GCNs) to explore connections between specific genes. 

The integration of multi-omics data may offer novel insights and provide a more comprehensive understanding of biological functions in the human body, in both health and disease. For example, a recent study integrated time series phenomics [[202118011006-Phenomics]], metabolomics, and fluxomics [[202118011007-Fluxomics]] data from subjects with various degrees of liver fat to understand more about non-alcoholic fatty liver disease (NAFLD). 

iNetModels is a user-friendly platform, providing exploratory capabilities and interactive/intuitive visualisations of clinical chemistry, anthropometric parameters, plasma proteins/metabolites, oral and gut microbome, associated with user-queried features. The data are obtained from recent studies, where large-scale MOBNs analyses were performed for individuals with different metabolic conditions.

The user can visualise features, download the network (table/figure), and export to Cytoscape. It also allows programmatic access to the database using a Python package.


![[Screenshot 2021-01-18 at 10.11.46.png]]

![[Screenshot 2021-01-18 at 10.12.09.png]]

#PhD #Permanent #MultiOmics #Cancer #Network #Bioinformatics #Transcriptomics #MoA 