#### Dose and Time Dependencies in Stress Pathway Responses during Chemical Exposure: Novel Insights from Gene Regulatory Networks
By: Terezinha M. Souza et al. (2017). Frontiers in Genetics. doi: 10.3389/fgene.2017.00142

Perturbation of biological networks often observed after exposure to xenobiotics [[202106010955-Xenobiotics]]. Disturbed processes, their dynamic traits, and dose-response relationships are challenging for elucidating the mechanisms of adverse outcomes. The usual strategy is to identify processes/pathways, and the second to estimate dose-response relationships that define the relationship between adaptive and adverse responses.

Here the authors performed a reverse engineering of gene regulatory networks (GRNs) from expression data, to provide a system-level snapshot with accurate molecular events. ODEs to establish causal link between external perturbations and gene-gene interactions within a particular biological process.

Groups of chemicals with distinct outcomes were investigated, carcinogenics (azathioprine and cyclophosphamide), and DILI (diclofenac, nitrofurantoin, and proplythiouracil), and non-carcinogenic/non-DILI (aspirin, diazepam and omeprazole). 

Publicly available human in vitro data was used, as well as dose- and time- dependencies. Dose-Time Network Identification (DTNI) was applied to gene sets from primary human hepatocytes using four stress pathways (ER, NF-kB, NRF2 and TP53). Dose–Time Network Identification (DTNI) is a method for inferring network interactions among genes through ODEs that relate changes in gene expression over time and dose and an external perturbation. DTNI requires measurements from multiple doses and time points – not necessarily sampled at equal intervals – and expression values of a reduced gene set (preferentially less than 100 genes). DTNI can be applied to single chemicals but also allows the use of group-wise approaches – in which a consensus network is inferred for multiple compounds. 

![[Screenshot 2021-01-06 at 10.05.15.png]]


#EPAProject #Permanent  #Bioinformatics #Transcriptomics #PhD 