#### Modelling Drug Response using Network-Based Personalised Treatment Prediction (NetPTP) With Applications to Inflammatory Bowel Disease
By: Licy Han, Zahra N. Sayyid and Russ B. Altman

Heterogeneity in disease (IBD) across patients makes it challenging to select optimal treatment. NetPTP models measured drug effects from gene expression data and applies them to patient samples to generate personalised ranked treatment lists. The protocl uses a combination of network, drug target and drug effect data with patient data. Ranked lists can then be used to prioritise existing treatments and discover new therapies for individual patients.

Representing disease as an aggregate can limit the methods in their ability to capture patient + disease heterogeneity. By treating each gene individually, methods fail to capture different combination of perturbations that cause similar disease phenotypes (contributing to disease heterogeneity). For complex diseases there are multiple avenues to target different aspects of the disease, and many patients do not respond to the same set of therapies. NetPTP incorporates interaction of genes + proteins w/ drug targets to repurpose and prioritize drugs in individual patients.

To construct the networks, CMap compounds + targets were extracted from DrugBank. Human pathways extracted from Reactome and connected in one large network  (only genes measured in CMap). Drug nodes added, connected to proteins/genes in the network.

The network was then parameterised by drugged gene expression data - each gene node is a linear regression of the incoming nodes and drug nodes are binary variables - "Diffusion of drug effect though gene network". Drugs can be turned "on and off" and the network captures changes induced by the effects.

The network was applied to IBD disease data to discover drugs that may be most effective for each individual patient. For each IBD sample, gene expression values were overlaid on the network. Drugs were then turned on or off, and the effects were propagated through the network - thus each IBD sample has a "drugged IBD sample" of gene expression. Using the healthy data, all healthy samples are averaged to create an "average healthy" gene expression sample. The average healthy sample was then compared to each drugged IBD sample (Euclidean distance) to create a personalised, ranked drug list. In essence, comparing the effects of different drugs to see which set can bring the original IBD sample closest to the average healthy sample - Also avoids side effects.

#PhD #Permanent #Bioinformatics #Network #Transcriptomics #MoA