#### Multiomics data integration unveils core transcriptional regulatory networks governing cell-type identity
By: Sascha Jung and Antonio del Sol
npj Systems Biology and Applications

Gene regulatory processes are often too complex to predict from transcriptomics data alone. To this end, Moni systematically integrates epigenetics, transcriptomics, and known PPIs to reconstruct a gene regulatory network (GRN), outperforming SOTA methods.

Cellular phenotypes are characterised by a stable gene expression profile, which is maintained by a set of TFs, determining the cell's identity. Together with other co-factors, the identity TFs form a regulatory "core" network, which is shaped by the epigenetic landscape. The epigenetic landscape is formed by epigenetic modifications, chromatin accessibility and chromatin conformation, and determines the phenotype-specific active regulatory regions of each core TF (thus controlling transcription rates of genes). Enhancer regions [[202115011631-BasicEpigenomicsNotes]] in particular modulate phenotype-specfic expression profiles. Transcriptomics data alone does not inform about active regulatory regions, thus hindering the ability to generate accurate GRNs.

Given a particular phenotype, Moni involves three main steps:
1. Core TFs are detected by comparing the expression of each TF to their expression in a background dataset of other cell types, from ArchS4
	- The essential determinants of cell identity are assigned to the top 10 TFs with the highest phenotypic specificity
	- Potential co-factors (TFs that are significantly more specific to the phenotype than their expected median specificty) are also identified in this step
2. Active promotors and ehancers of core TFs and co-factors identified [[202115011631-BasicEpigenomicsNotes]]
	- Promotor region are those that overlap with at least one H3K4me3 peak
	- Enhancer regions are associated to TFs based on GeneHancer, and are assigned as active if they overlap with at least one H3K27ac peak
3. The directed interactions among core TFs/co-factors are inferred if they satisfy the following conditions:
	- Promotor of the TF is active
	- Interaction supported by a non-phenotype-specific ChIP-seq peak in promoter or active enhancer region of target TF
	- Supporting ChIP-seq peak falls within accessible chromatin region

They validated the approach in three well-characterised cell lines using this workflow:

![[Screenshot 2021-01-15 at 16.40.19.png]]

#PhD #Permanent #Transcriptomics #Network #MoA #Bioinformatics #Cancer #MultiOmics #Epigenomics 