#### Drug perturbation gene set enrichment analysis (dpGSEA): a new transcriptomic drug screening approach
By: Mike Fang et al.
BMC Bioinformatics

This study outlays a novel GSEA method called drug perturbation GSEA (dpGSEA), which can detect phenotypically relevant drug targets through a unique transcriptomic enrichment that emphasises biological directionality of drug-derived gene sets. They demonstrate the approach using fluvastatin, paclitaxel and rosiglitazone perturbations, and it was able to detect phenotypically relevant drug targets.

They base their work on the signature reversion principles - their premise is that a negatively correlated gene profile of a drug-perturbated transcriptome can be expoited for in silico drug screening. By performing enrichment on disease-associated gene signatures while using drug perturbation defined gene sets, entire transcriptomes can be probed for potential drugs and therapeutics. dpGSEA hence performs a unique drug-defined gene set enrichment analysis for screening therapeutics downstream of transcriptomic/proteomic studies.

![[Screenshot 2021-01-19 at 09.07.29.png]]

![[Screenshot 2021-01-19 at 09.07.50.png]]

dpGSEA utilises transcriptomic signatures of drug perturbed cell lines from CMAP/LINCS to produce annotated gene sets rather than curated lists (such as those from MSigDB). These are organised into proto-matrices, as defined by gene signature cutoffs of ranked top fold change/statistical significance. The proto-matrix itself contains information including genes acted on by a specific drug, and the directionality in which it is influenced (up or down regulation). 

dpGSEA detects an enrichment of differentially expressed genes in the gene sets. This tranlates into finding evidence of a leading-edge subset in the gene set, in which the expression of the genes are maximal.

Through this work they were able to find highly enriched CMap/LINCS gene sets, from which they could discover targets.

#PhD #Permanent #Transcriptomics #Bioinformatics #MoA 